-Wfortify-source warnings

While compiling 0.8.4.1 I get:

```
 * QA Notice: Package triggers severe warnings which indicate that it
 *            may exhibit random runtime failures.
 * /tmp/portage/net-analyzer/ettercap-0.8.4.1/work/ettercap-0.8.4.1/src/ec_scan.c:691:133: warning: 'fscanf' may overflow; destination buffer in argument 3 has size 47, but the corresponding specifier may require size 48 [-Wfortify-source]
 * /tmp/portage/net-analyzer/ettercap-0.8.4.1/work/ettercap-0.8.4.1/src/ec_scan.c:691:137: warning: 'fscanf' may overflow; destination buffer in argument 4 has size 19, but the corresponding specifier may require size 20 [-Wfortify-source]
 * /tmp/portage/net-analyzer/ettercap-0.8.4.1/work/ettercap-0.8.4.1/src/ec_scan.c:691:142: warning: 'fscanf' may overflow; destination buffer in argument 5 has size 64, but the corresponding specifier may require size 65 [-Wfortify-source]
```

--- a/src/ec_scan.c
+++ b/src/ec_scan.c
@@ -670,9 +670,9 @@ int scan_load_hosts(char *filename)
 {
    FILE *hf;
    int nhosts;
-   char ip[MAX_ASCII_ADDR_LEN];
-   char mac[ETH_ASCII_ADDR_LEN];
-   char name[MAX_HOSTNAME_LEN];
+   char ip[MAX_ASCII_ADDR_LEN+1];
+   char mac[ETH_ASCII_ADDR_LEN+1];
+   char name[MAX_HOSTNAME_LEN+1];
    struct ip_addr hip;
    u_int8 hmac[MEDIA_ADDR_LEN];
 

